reconcile/utils/jira_client.py

```diff
@@ -15,21 +18,41 @@ def __init__(self, jira_board, settings=None):
         token_auth = self.secret_reader.read(token)
         self.jira = JIRA(self.server, token_auth=token_auth)
 
-    def get_issues(self, fields=None):
+    def get_issues(self, fields: Optional[Mapping] = None) -> list:
         block_size = 100
         block_num = 0
-        all_issues = []
+
+        all_issues = []  # type: ignore
         jql = 'project={}'.format(self.project)
         kwargs = {}
         if fields:
             kwargs['fields'] = ','.join(fields)
         while True:
             index = block_num * block_size
             issues = self.jira.search_issues(jql, index, block_size,
-                                             **kwargs)
+                                             **kwargs)  # type: ignore
             all_issues.extend(issues)
             if len(issues) < block_size:
                 break
             block_num += 1
 
         return all_issues
+
+    def create_issue(self, summary: str, body: str,
+                     labels: Optional[Iterable[str]] = None,
+                     links: Iterable[str] = ()) -> Issue:
```

Comment:
Does this not suffer from [this issue](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) because tuples aren't mutable?

Reply:
Exactly.